app/code/Magento/InventoryInStorePickupApi/Api/Data/SearchResultInterface.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryInStorePickupApi\Api\Data;
+
+use Magento\InventoryInStorePickupApi\Api\Data\SearchCriteria\GetNearbyLocationsCriteriaInterface;
+
+/**
+ * Search results for providing nearby pickup locations
+ */
+interface SearchResultInterface
+{
+    /**
+     * Get items list.
+     *
+     * @return PickupLocationInterface[]
+     */
+    public function getItems(): array;
+
+    /**
+     * Set items list.
+     *
+     * @param PickupLocationInterface[] $items
+     *
+     * @return $this
+     */
+    public function setItems(array $items);
+
+    /**
+     * Get search criteria.
+     *
+     * @return GetNearbyLocationsCriteriaInterface
+     */
+    public function getSearchCriteria(): GetNearbyLocationsCriteriaInterface;
```

Comment:
I am not sure about this.
We will have additional Search Criteria for endpoint which return list of Pickup Locations with filters.
It will be nice to have one Search Result for both cases.
Probably we need some general Search Criteria interface which will be inherited by certain Search Criteria for endpoint.
@ishakhsuvarov what do you think about this?

Reply:
I just did it.